presentation/src/main/AndroidManifest.xml

```diff
@@ -14,10 +14,11 @@
         android:supportsRtl="true"
         android:theme="@style/AppTheme"
         tools:ignore="AllowBackup,GoogleAppIndexingWarning">
-
+        <activity android:name=".events.EventsActivity"></activity>
```

Comment:
Put this below with the other activities and close the activity tag at the end of the name like this `<activity android:name=".events.EventsActivity"/>`

Reply:
Done